Lib/test/test_perf_profiler.py

```diff
@@ -35,17 +35,17 @@ def supports_trampoline_profiling():
 
 
 class TestPerfTrampoline(unittest.TestCase):
-    def setUp(self):
-        super().setUp()
-        self.perf_files = set(pathlib.Path("/tmp/").glob("perf-*.map"))
-
-    def tearDown(self) -> None:
-        super().tearDown()
-        files_to_delete = (
-            set(pathlib.Path("/tmp/").glob("perf-*.map")) - self.perf_files
-        )
-        for file in files_to_delete:
-            file.unlink()
+
+    def _cleanup_perf_map(self, pid):
+        """
+        Helper to safely delete a specific perf map file.
+        """
+        perf_map = pathlib.Path(f"/tmp/perf-{pid}.map")
+        try:
+            if perf_map.exists():
+                perf_map.unlink()
+        except OSError:
+            pass
```

Comment:
```suggestion
        os_helper.unlink(f"/tmp/perf-{pid}.map")
```

I think this should be enough but can you if (test.support.)os_helper.unlink() suppresses OS errors?